.evergreen/start-orchestration.sh

```diff
@@ -59,7 +59,7 @@ else
   fi
 fi
 
-nohup mongo-orchestration $ORCHESTRATION_ARGUMENTS start > $MONGO_ORCHESTRATION_HOME/out.log 2> $MONGO_ORCHESTRATION_HOME/err.log < /dev/null &
+set -o pipefail && mongo-orchestration $ORCHESTRATION_ARGUMENTS start < /dev/null | tee $MONGO_ORCHESTRATION_HOME/out.log
```

Comment:
This change will make it easier to diagnose `mongo-orchestration start` failures because:
- we now fail fast when `start` fails
- we log the `start` output to stdout in addition to `out.log`
- we don't need to use `&` because `start` launches a daemon process

Reply:
Turns out we do need to use `&` or else the `start` command hangs. I'm not sure why and I don't want to find out. We also can't use pipefail on RHEL 7.0 (I think we could change these scripts to use `/bin/bash` instead of `/bin/sh` but that might open a whole new can of worms. Instead I changes this back to use `&` but I did add some better logging when MO fails to start (or fails to start a cluster).